### PR TITLE
add ability to hide the target

### PR DESCRIPTION
**Pull request by Raynos**
This allows us to have three targets, two hidden graphs and
a `divideSeries(#A,#B)` which is useful for dividing two
non trivial series.

r: @MadanThangavelu @eculver

### CODE REVIEW

**Comment by eculver** [comment]
Mind adding a test to preserve our 100% coverage?

File: grafana/target.js [hunk 1]
```diff
@@ -204,4 +204,9 @@ Target.prototype.summarize15min = function summarize15min() {
     return this.summarize('15min');
 };
 
+Target.prototype.hide = function hide() {
+    this.hide = true;
```
Comment (eculver) [inline grafana/target.js]: Actually, this won't work. When the panels are serialized, the `hide` property needs to be set there, not on the target.

**Comment by Raynos** [comment]
I also changed the semantics of `current` and `options` in templates such that the first option is the current;

In my head and the old graphite-dash-gen it makes sense to default to the HEAD instead of the TAIL of the list.

**Comment by eculver** [comment]
Solid. Thanks for removing cached-tape too. I don't mind the semantic change either with regards to templates. It shouldn't cause any incompatible changes.

**Pull request merged**